prime-router/src/test/kotlin/fhirengine/azure/FhirFunctionIntegrationTests.kt

```diff
@@ -383,6 +462,303 @@ class FhirFunctionIntegrationTests() {
         }
         verify(exactly = 1) {
             QueueAccess.sendMessage(elrRoutingQueueName, any())
+            BlobAccess.uploadBody(Report.Format.FHIR, any(), any(), any(), any())
+        }
+    }
+
+    @Test
+    fun `test successfully processes a convert message for bulk HL7 message`() {
+        val validBatch = cleanHL7Record + "\n" + invalidHL7Record
```

Comment:
So this batch's second item is invalid but it still goes through because the particular record its not considered invalid by the parser?

Reply:
Yep "invalid" in this case means having a segment that does not exist in ORU_R01